content/docs-v1/collection-catalogues.md

```diff
@@ -20,107 +68,201 @@ object_display_order:
+
+If the object figure is not set to `zoom: true`, you will need to include both the figure `id` and the `thumbnail` path.
+
+```yaml
+object_list:
+  - id: 1
+    artist:
+    year:
+    dimensions:
+    medium:
+    location:
+    thumbnail: 
+    figure: 
+      - id: "fig-1-1"
+      - id: "fig-1-2"
+      - id: "fig-1-3"
 ```
 
-{{< q-figure id="modern-entry-side" >}}
+## Generate Object Grids
 
-Whatever the presentation, you can further organize the content on the page by using Quire’s `{% accordion %}` shortcode to wrap the individual parts of the object entry into collapsible sections. Read more in the [*Accordions*](/docs-v1/accordions/) chapter of this guide.
+In a collection catalogue, there is typically a visual table of contents that unifies all the catalogue entries/objects. 
 
-{{< q-figure id="modern-entry-side-accordion" >}}
+[Add object grid screenshot]
 
+To create this visual table of contents, which we refer to as an object grid there are three key steps. Add `layout: object-grid` to a `.md` file, add an `object-card` list to your `objects.yaml` file, and add an `object-filters` list to the `objects.yaml` file.
 
-### Object Images
+{{< q-class "box warning" >}}
+- As described in the [*Page Types & Structure*](/docs-v1/pages/#create-section-landing-pages) section of the documentation, an `index.md` file is used within a subfolder to create a landing page for all the contents found in that directory. An approximation of an object grid can be created by adding `layout: table-of-contents` and `presentation: grid` to the `index.md file`. However, we recommend reserving `layout: table-of-contents` for `essay` pages only.
+{{< /q-class >}}
 
-You can add more than one image to your object entry page by listing multiple figure `ids` in your `objects.ymal` file. Multiple images are displayed as a rotating carousel and are arranged in the order they are listed in the `objects.yaml` file. If any of the object images have a `caption` and/or `credit`, that information will be included as a pop-up window.
+### Step 1: Add `layout: object-grid` to a designated Markdown file 
 
-If the figure’s `download` attribute is set to `true`, a download icon will be included as well.
+You can add `layout: object-grid` to any `.md` file, whether it's an `index.md` file in a subdirectory or a regular file in the root directory. 
 
-### Object Link
+```yaml
+layout: object-grid
+```
 
-If the `object_list` includes a `link` value, a “View in Collection” button is generated. The text of this button can be customized with the `objectLinkText` attribute in the project’s `content/_data/config.yaml` file.
+Similar to an `index.md` file, nothing else needs to appear below the page YAML on that page. The contents will be automatically populated. 
```

Comment:
I don't think you mean "`index.md` file" so much as `layout: table-of-contents`?